### PR TITLE
🔀🌐 [pl_PL] add translation for missionHelper

### PR DESCRIPTION
**Pull request by Kemon22**
**What kind of update does this PR provide?**
- [x] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *...*

**Which language(s) did you add/update translations for?**
* pl_PL

**Additional notes**
Sorry for that small PR

### CODE REVIEW

File: src/modules/missionHelper/i18n/pl_PL.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
             "height_rescue_units": "SRWys",
             "helicopter": "LPR",
             "hose": "SW",
-            "k9": "K-9 | K-9",
+            "k9": "K-9",
```
Comment (Cr4zyc4k3) [inline src/modules/missionHelper/i18n/pl_PL.json:73]: No plural?

File: src/modules/missionHelper/i18n/pl_PL.json [hunk 2]
```diff
@@ -86,24 +86,27 @@
...
+            },
+            "allow_ktw_instead_of_rtw": {
+                "ambulances": "Ambulans | Ambulanse"
             },
             "allow_rw_instead_of_lf": {
-                "firetrucks": "Wóz strażacki | Wozy strażackie"
+                "firetrucks": "Samochód pożarniczy lub ratownictwa technicznego | Samochody pożarnicze lub ratownictwa technicznego"
+            },
+            "allow_dlk_instead_of_lf": {
+                "firetrucks": "Samochód pożarniczy lub SD/SH | Samochody pożarnicze lub SD/SH"
```
Comment (Cr4zyc4k3) [inline src/modules/missionHelper/i18n/pl_PL.json:109]: @jxn-30 Does a `/` work without escaping?

Comment (jxn-30) [reply src/modules/missionHelper/i18n/pl_PL.json:109]: yes.

File: src/modules/missionHelper/i18n/pl_PL.json (hunk 1 above)
Comment (Kemon22) [reply src/modules/missionHelper/i18n/pl_PL.json:73]: If it is not required then it is ok :P

**Review by Cr4zyc4k3 — APPROVED** [review]

**Pull request merged**